pkg/reconciler/pipelinerun/pipelinerun_updatestatus_test.go

```diff
@@ -41,168 +44,119 @@ type updateStatusTaskRunsData struct {
 	simple         map[string]*v1beta1.PipelineRunTaskRunStatus
 }
 
-func getUpdateStatusTaskRunsData() updateStatusTaskRunsData {
-	// PipelineRunConditionCheckStatus recovered by updatePipelineRunStatusFromTaskRuns
-	// It does not include the status, which is then retrieved via the regular reconcile
-	prccs2Recovered := map[string]*v1beta1.PipelineRunConditionCheckStatus{
-		"pr-task-2-running-condition-check-xxyyy": {
-			ConditionName: "running-condition-0",
-		},
-	}
-	prccs3Recovered := map[string]*v1beta1.PipelineRunConditionCheckStatus{
-		"pr-task-3-successful-condition-check-xxyyy": {
-			ConditionName: "successful-condition-0",
-		},
+func getUpdateStatusTaskRunsData(t *testing.T) updateStatusTaskRunsData {
```

Comment:
cleanup like swapping over to yaml parsing should be in a separate PR, although OK if you don't want to do it this time since the original PR has already been split several times

Reply:
I thought it made sense to just do it here. =)

Reply:
(also I originally did the ->YAML parsing change before I split this and the remainder of the implementation up, so it was markedly simpler to just bring it along)